### PR TITLE
Add release notes for rc1

### PR DESCRIPTION
**Pull request by gaiksaya**
Signed-off-by: Sayali Gaikawad <gaiksaya@amazon.com>

### Description
Adds release notes for rc1 release
 
### Issues Resolved
None
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-release-notes-1.0.0-rc1.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+
+## FEATURES
+
+### Opensearch Index Management
+* Adds support for Transform feature ([#17](https://github.com/opensearch-project/index-management/pull/17))
+
+
+### Opensearch Index Management Dashboards Plugin
+* Introducing transforms ([#16](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/16))
+* Show data streams in 'Indices' and 'Managed Indices' sections ([#24](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/24))
```
Comment (bbarani) [inline release-notes/opensearch-release-notes-1.0.0-rc1.md:19]: I think this PR is not part of this release?

File: release-notes/opensearch-release-notes-1.0.0-rc1.md [hunk 2]
```diff
@@ -0,0 +1,334 @@
+# OpenSearch and Dashboards 1.0.0-rc1 Release Notes
+
+## Release Highlights
+
+## Release Details
+
+OpenSearch and Dashboards 1.0.0-rc1 includes the following OpenSearch Migration, features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+## BREAKING CHANGES
```
Comment (elfisher) [inline release-notes/opensearch-release-notes-1.0.0-rc1.md:9]: Let's remove this section.

File: release-notes/opensearch-release-notes-1.0.0-rc1.md [hunk 3]
```diff
@@ -0,0 +1,334 @@
+# OpenSearch and Dashboards 1.0.0-rc1 Release Notes
+
+## Release Highlights
```
Comment (elfisher) [inline release-notes/opensearch-release-notes-1.0.0-rc1.md:3]: Suggested text:

This Release Candidate is a version of the project that is feature complete and passing automated testing with the intent to validate expected functionality before moving to a General Availability (GA) launch. There are multiple enhancements and fixes that are part of this release including span filtering support in Trace Analytics, tenant support in Notebooks, K-NN field level algorithm selection, support for index management transforms, and support for scheduling and tenants in reporting.

**Review by elfisher — CHANGES_REQUESTED** [review]

File: release-notes/opensearch-release-notes-1.0.0-rc1.md (hunk 1 above)
Comment (gaiksaya) [reply release-notes/opensearch-release-notes-1.0.0-rc1.md]: Removed

File: release-notes/opensearch-release-notes-1.0.0-rc1.md (hunk 2 above)
Comment (gaiksaya) [reply release-notes/opensearch-release-notes-1.0.0-rc1.md]: Removed

File: release-notes/opensearch-release-notes-1.0.0-rc1.md (hunk 3 above)
Comment (gaiksaya) [reply release-notes/opensearch-release-notes-1.0.0-rc1.md:3]: Added. Thanks

**Review by elfisher — APPROVED** [review]
LGTM thanks @gaiksaya!

File: release-notes/opensearch-release-notes-1.0.0-rc1.md [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+
+
+### Opensearch Security
+* Allow attempt to load security config in case of plugin restart even if security index already exists ([#1154](https://github.com/opensearch-project/security/pull/1154))
+* Allowing granular access for data-stream related transport actions ([#1170](https://github.com/opensearch-project/security/pull/1170))
+
+
+### Opensearch Security Dashboards Plugin
+* Add svg content [#766](https://github.com/opensearch-project/security-dashboards-plugin/pull/766)
+* Changes regarding the anonymous auth flow [#768](https://github.com/opensearch-project/security-dashboards-plugin/pull/768)
```
Comment (sreekarjami) [inline release-notes/opensearch-release-notes-1.0.0-rc1.md:73]: PR info not enclosed in (), not following the same standard as others.

**Review by camerski — APPROVED** [review]

File: release-notes/opensearch-release-notes-1.0.0-rc1.md [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+
+
+### Opensearch Security Dashboards Plugin
+* move issue templates to ISSUE_TEMPLATE [#758](https://github.com/opensearch-project/security-dashboards-plugin/pull/758)
+* Update issue template with multiple labels [#756](https://github.com/opensearch-project/security-dashboards-plugin/pull/756)
+* Naming convention change in repo [#764](https://github.com/opensearch-project/security-dashboards-plugin/pull/764)
+* Point document links to opensearch.org [#763](https://github.com/opensearch-project/security-dashboards-plugin/pull/763)
+* Create release notes for rc-1 and Bump version to 1.0.0.0-rc1 [#771](https://github.com/opensearch-project/security-dashboards-plugin/pull/771)
+* Point OpenSearch Dashboards to to new back end APIs [#770](https://github.com/opensearch-project/security-dashboards-plugin/pull/770)
+* Change the titles of release notes [#772](https://github.com/opensearch-project/security-dashboards-plugin/pull/772)
```
Comment (sreekarjami) [inline release-notes/opensearch-release-notes-1.0.0-rc1.md:290]: PR info not enclosed in (), not following the same standard as others.

File: release-notes/opensearch-release-notes-1.0.0-rc1.md [hunk 6]
```diff
@@ -0,0 +1,324 @@
...
+
+
+### Opensearch Security
+* Delay the security index initial bootstrap when the index is red ([#1153](https://github.com/opensearch-project/security/pull/1153))
+* Remove redundant isEmpty check and incorrect string equals operator ([#1181](https://github.com/opensearch-project/security/pull/1181))
+
+
+### Opensearch Security Dashboards Plugin
+* Fixing JSON parsing in SAML for strings that contain '' character [#749](https://github.com/opensearch-project/security-dashboards-plugin/pull/749)
+* Fix login redirect [#777](https://github.com/opensearch-project/security-dashboards-plugin/pull/777)
```
Comment (sreekarjami) [inline release-notes/opensearch-release-notes-1.0.0-rc1.md:115]: PR info not enclosed in (), not following the same standard as others.

File: release-notes/opensearch-release-notes-1.0.0-rc1.md (hunk 4 above)
Comment (gaiksaya) [reply release-notes/opensearch-release-notes-1.0.0-rc1.md]: Added. Thanks @sreekarjami

File: release-notes/opensearch-release-notes-1.0.0-rc1.md (hunk 6 above)
Comment (gaiksaya) [reply release-notes/opensearch-release-notes-1.0.0-rc1.md]: Added

File: release-notes/opensearch-release-notes-1.0.0-rc1.md (hunk 5 above)
Comment (gaiksaya) [reply release-notes/opensearch-release-notes-1.0.0-rc1.md]: Done

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**